docs/elasticsearch-cluster.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: operator
+  namespace: es-operator-demo
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: es-config
+  namespace: es-operator-demo
+data:
+  elasticsearch.yml: |
+    cluster.name: es-cluster
+    network.host: "0.0.0.0"
+    bootstrap.memory_lock: false
+    discovery.zen.ping.unicast.hosts: es-cluster
+    discovery.zen.minimum_master_nodes: 1
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: es-master
+  namespace: es-operator-demo
+  annotations:
+    downscaler/exclude: "true"
```

Comment:
I think this [kube-downscaler](https://github.com/hjacobs/kube-downscaler) annotation is not relevant for this tutorial, remove?